.vib/oauth2-proxy/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/oauth2-proxy"
         },
-        "runtime_parameters": "InJlZGlzIjoKICAiYXV0aCI6CiAgICAicGFzc3dvcmQiOiAiam5kZkRGZiIKInNlcnZpY2UiOgogICJwb3J0IjogODAKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIi",
+        "runtime_parameters": "c2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0OiA4MApjb25maWd1cmF0aW9uOgogIGNsaWVudElEOiB2aWItdGVzdHMtaWQKICBjbGllbnRTZWNyZXQ6IHZpYi10ZXN0cy1zZWNyZXQKICBjb29raWVTZWNyZXQ6IHZpYi10ZXN0cy1jb29raWUKICBjb250ZW50OiB8CiAgICBlbWFpbF9kb21haW5zID0gWyAiKiIgXQogICAgdXBzdHJlYW1zID0gWyAiZmlsZTovLy9kZXYvbnVsbCIgXQpjb250YWluZXJQb3J0OiA0MTgxCnBvZFNlY3VyaXR5Q29udGV4dDoKICBlbmFibGVkOiB0cnVlCiAgZnNHcm91cDogMTAwMgpjb250YWluZXJTZWN1cml0eUNvbnRleHQ6CiAgZW5hYmxlZDogdHJ1ZQogIHJ1bkFzVXNlcjogMTAwMgpzZXJ2aWNlQWNjb3VudDoKICBjcmVhdGU6IHRydWUKICBhdXRvbW91bnRTZXJ2aWNlQWNjb3VudFRva2VuOiB0cnVlCnJlZGlzOgogIGVuYWJsZWQ6IHRydWUKICBhcmNoaXRlY3R1cmU6IHN0YW5kYWxvbmUKICBhdXRoOgogICAgZW5hYmxlZDogdHJ1ZQogICAgcGFzc3dvcmQ6IG9hdXRoMi12aWItcGFzc3dvcmQ=",
```

Comment:
After checking locally, I do not see any difference between having `redis.architecture` ad `standalone/replication` so I would say you can drop that param from here.